scripts/lst-rfperformance.py

```diff
@@ -21,13 +21,19 @@
 parser = argparse.ArgumentParser(description = "Train Random Forests.")
 
 # Required argument
+'''
 parser.add_argument('--gammafile', '-fg', type=str,
                     dest='gammafile',
                     help='path to the dl1 file of gamma events for training')
 
 parser.add_argument('--protonfile', '-fp', type=str,
                     dest='protonfile',
                     help='path to the dl1 file of proton events for training')
+'''
+parser.add_argument('--inpath', '-i', action='store', type=str,
```

Comment:
why this change of API?

Reply:
Hi Thomas! Thanks for the review of the notebook.
Ah, the other commits seems to be things I did  after the bootcamp due to some comments I received. 
The change in the RFperformance.py was in order to always use the same train and test events, instead of building them randomly each time the script is ran.

Reply:
I see. But this just moves the issue from pointing to training files to a directory where train/test splitting has already been applied, no?
I think a more versatile (and still easy and consistent) way would be to do the train/validation splitting in the script with a default random seed so that we know the training dataset is always the same when running for the same file.

In any case, could you limit the PR to its intended purpose and have only the notebook? It's better to have multiple PR for different purposes so we can discuss changes separately.
Thank you.

Reply:
Thank you.
PR can be merged I think.